tests/browser/interactionHelpers.ts

```diff
@@ -0,0 +1,20 @@
+import { test } from '@playwright/test';
+import themes from '../../themes.json';
+
+export { expectScreenshot } from './expectScreenshot';
+
+export function sleep(ms) {
```

Comment:
Nice, glad to see a robust set of shared utility functions for tests.

What do you think about running a search in the codebase for `const sleep` or function sleep` and replacing those inline functions with this shared function?

Reply:
The other local definitions for sleep functions that I can find are in unit tests. Are you suggesting we share a helper module between browser tests and unit tests?

Reply:
Thanks for pointing that out. No, let's not create a helper module between those types of tests. That work's outside the scope of this PR.